### PR TITLE
fix: remove mkr and fallback to the default synth when base/quote is invalid

### PR DESCRIPTION
**Pull request by evgenyboxer**

### CODE REVIEW

File: src/ducks/synths.ts [hunk 1]
```diff
@@ -99,7 +97,7 @@ export const synthsSlice = createSlice({
 			const availableSynths: SynthDefinitionMap = keyBy(
 				synths.map((synth) => ({
 					...synth,
-					isFrozen: FROZEN_SYNTHS.includes(synth.name),
+					isFrozen: false,
```
Comment (evgenyboxer) [inline src/ducks/synths.ts:100]: by default all synths are not frozen, and then updated as soon as we get them from the frozen list

File: src/pages/Trade/Trade.tsx [hunk 2]
```diff
@@ -18,30 +18,40 @@ import CreateOrderCard from './components/CreateOrderCard';
...
+	setSynthPair,
 };
 
-type Props = RouteComponentProps<{
-	baseCurrencyKey: CurrencyKey;
-	quoteCurrencyKey: CurrencyKey;
-}>;
+const connector = connect(mapStateToProps, mapDispatchToProps);
+
+type PropsFromRedux = ConnectedProps<typeof connector>;
```
Comment (evgenyboxer) [inline src/pages/Trade/Trade.tsx:34]: @clementbalestrat this is the new pattern for typing redux ^ so much shorter and better!

File: src/pages/Trade/Trade.tsx [hunk 3]
```diff
@@ -18,30 +18,40 @@ import CreateOrderCard from './components/CreateOrderCard';
...
 	useEffect(() => {
 		const { params } = match;
 
-		if (params && params.baseCurrencyKey && params.quoteCurrencyKey) {
+		if (
+			params &&
+			params.baseCurrencyKey &&
+			params.quoteCurrencyKey &&
+			synthsMap[params.baseCurrencyKey] &&
+			synthsMap[params.quoteCurrencyKey]
```
Comment (evgenyboxer) [inline src/pages/Trade/Trade.tsx:53]: ensure base/quote exists in the synthsMap - in not, redirect to our default synths

File: src/utils/synthOrdering.js [hunk 4]
```diff
@@ -7,7 +7,6 @@ export const synthWeight = {
 	XTZ: 6,
 	TRX: 7,
 	LINK: 8,
-	MKR: 9,
```
Comment (evgenyboxer) [inline src/utils/synthOrdering.js:10]: removed all instances of "MKR"

File: src/pages/Trade/Trade.tsx (hunk 2 above)
Comment (clementbalestrat) [reply src/pages/Trade/Trade.tsx:34]: oh cool, love it

**Pull request merged**